cypress/integration/t2.js

```diff
@@ -0,0 +1,35 @@
+describe('User roles', function() {
+        
+    it('Should display validation for empty user after attempted loggin', function () {
+        cy.visit('/')
+        cy.get('.Select.not-valid').should('not.visible')
+        cy.get('[type="submit"]').click()
+        cy.get('.Select.not-valid').should('be.visible')
+    })
+
+    it('Login with all different roles and do some check', function () {
+        var roles = ["User", "Team Lead", "Manager", "Accountant", "Admin" ];        
+        var tabs = [1,2,5,5,6];        
+        roles.forEach(checkAllRoles);
+
+        function checkAllRoles(value, index, array) {
+
+        cy.get('[id="loginForm.userId"]').click({force:true})
+        cy.get('[aria-label="TestCon User 7"]').click()
+        cy.get('[id="loginForm.role"]').click({force:true})
+        cy.get('[aria-label="' + value + '"').click()
+
+        // log in
```

Comment:
You don't actually have to add any comments. It is best to write such code that is understandable even without comments :)